src/mockClient.ts

```diff
@@ -11,14 +11,15 @@ export type RequestHandlerResponse<T> = { data: T, errors?: any[] };
 export type MockApolloClient = ApolloClient<NormalizedCacheObject> &
   { setRequestHandler: (query: DocumentNode, handler: RequestHandler) => void };
 
-export const createMockClient = (): MockApolloClient => {
+export const createMockClient = (options?: Partial<ApolloClientOptions<NormalizedCacheObject>>): MockApolloClient => {
```

Comment:
I don't think it should be possible to supply link, otherwise the mock apollo client won't work as expected.
Creating a new type might make it more readable?

```suggestion
export type MockApolloClientOptions = Partial<Omit<ApolloClientOptions<NormalizedCacheObject>, 'link'>>;

export const createMockClient = (options?: MockApolloClientOptions): MockApolloClient => {
```